Web/Views/Home/_partials/Banner.cshtml

```diff
@@ -15,14 +15,14 @@
             switch (selector)
             {
                 case 1:
-                    <a href="https://bit.ly/xam-dev"><img class="img-fluid" src="~/img/banners/megsoft-job.png" /></a>
+                    <a href="https://bit.ly/xam-dev" target="_blank()"><img class="img-fluid" src="~/img/banners/megsoft-job.png" /></a>

```

Comment:
Hi Junior @jsuarezwd , the `target` attribute should have the value `_blank` instead of `_blank()`, also I consider we should include the attribute `rel` with the values `noopener noreferrer` for security reasons. See suggested changes:

```suggestion
                    <a href="https://bit.ly/xam-dev" target="_blank" rel="noopener noreferrer"><img class="img-fluid" src="~/img/banners/megsoft-job.png" /></a>

```
Can you please apply those changes to each banner?

Source:
- [HTML \<a> target Attribute](https://www.w3schools.com/tags/att_a_target.asp)
- [HTML \<a> rel Attribute](https://www.w3schools.com/tags/att_a_rel.asp)
- [target="_blank" is a security risk?](https://dev.to/thehassantahir/targetblank-is-a-security-risk-1ee4)